File: Go/001/main.go
package main

import (
	"fmt"
	"time"
)

const (
	limit = 1_000_000
)

func solve(to int) int {
	var answer int

	for i := 3; i < to; i++ {
		if i%3 == 0 || i%5 == 0 {
			answer += i
		}
	}

	return answer
}

func main() {
	start := time.Now()
	answer := solve(limit)
	elapsed := time.Since(start)

	fmt.Println(answer)
	fmt.Printf("%.6f seconds\n", elapsed.Seconds())
}

File: Julia/001/main.jl
const limit = 1_000

function solve(to)
    answer = 0

    for i in 3:to-1
        if i % 3 == 0 || i % 5 == 0
            answer += i
        end
    end

    return answer
end

println(solve(limit))
@time solve(limit)
